jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/SchemaGeneratorConfigBuilder.java

```diff
@@ -306,4 +309,18 @@ public SchemaGeneratorConfigBuilder withAnnotationInclusionOverride(Class<? exte
         this.annotationInclusionOverrides.put(annotationType, override);
         return this;
     }
+
+    /**
+     * Register a custom {@link ObjectMapper} to use for serialization of the schema. This can be used to customize
+     * the serialization behavior of the schema (eg serialize to yaml instead of json).
+     *
+     * @param objectMapper the {@link ObjectMapper} to use for serializing schemas
```

Comment:
**suggestion**: Let's mention the main purpose of the `ObjectMapper` and not just the scenario mostly relevant to the Maven plugin (as stated on the constructors).
```suggestion
     * Register a custom {@link ObjectMapper} to create object and array nodes for the JSON structure being
     * generated. Additionally, it is used to serialize a given schema, e.g., within the standard Maven plugin as
     * either YAML or JSON.
     *
     * @param objectMapper supplier for object and array nodes for the JSON structure being generated
```